WordPress/src/androidTest/java/org/wordpress/android/e2e/StatsTests.java

```diff
@@ -0,0 +1,49 @@
+package org.wordpress.android.e2e;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.wordpress.android.e2e.pages.MySitesPage;
+import org.wordpress.android.support.BaseTest;
+import org.wordpress.android.util.StatsKeyValueData;
+import org.wordpress.android.util.StatsMocksReader;
+import org.wordpress.android.util.StatsVisitsData;
+
+import java.util.List;
+
+public class StatsTests extends BaseTest {
+    @Before
+    public void setUp() {
+        logoutIfNecessary();
+        wpLogin();
+    }
+
+    @After
+    public void tearDown() {
+    }
+
+    @Test
+    public void allDayStatsLoad() {
+        StatsVisitsData todayVisits = new StatsVisitsData("97", "28", "14", "11");
```

Comment:
I went somewhat against the "isolate test data from test code" principle in the case with the [Day Visits Mock file](https://github.com/wordpress-mobile/WordPress-Android/blob/adds-stats-ui-test/libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/stats/stats_visits_unit_day_12.json), since the file does not have an intuitive and easy-to-parse structure, and the effort with this particular file would most probably not worth it. Instead, I went for the good 'ol test data hardcoding.

Reply:
Hey, I tested it locally and it looks like the actual data is coming from another file. I don't see the hardcoded as a show stopper but it would definitely help with consistency.
https://github.com/wordpress-mobile/WordPress-Android/blob/0b32bc90961cec597554e19360ce7bac0b93a9fe/libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/stats/stats_visits_unit_day_15.json#L41